themes/opentermsarchive/layouts/_default/about.html

```diff
@@ -73,30 +73,52 @@ <h3>{{ i18n "about.funders.title" }}</h3>
           <p class="text--light mb--3xl">{{ i18n "about.funders.details" | markdownify }}</p>
         </div>
         <div>
-          <h3>{{ i18n "about.core.title" }}</h3>
-          <p>{{ i18n "about.core.subtitle" }}</p>
-          {{ partial "contributors.html" (dict "staffType" "core" "thumbnailWidth" 88 "thumbnailHeight" 88 "showInfo" true "class" "contributors--has-no-margin-top mb--xl" ) }}
+          {{ partial "contributors.html" (dict
+            "staffType" "core"
+            "thumbnailWidth" 88
+            "thumbnailHeight" 88
+            "showInfo" true
+            "class" "contributors--has-no-margin-top mb--xl"
+            "title" (i18n "about.core.title")
+            "description" (i18n "about.core.subtitle")
+            ) }}
         </div>
         <div>
-          <h4>{{ i18n "about.alumni.title" }}</h4>
-          <p>{{ i18n "about.alumni.subtitle" }}</p>
-          {{ partial "contributors.html" (dict "staffType" "alumni" "showInfo" true "class" "mb--2xl contributors--has-no-margin-top" ) }}
+          {{ partial "contributors.html" (dict
+            "staffType" "alumni"
+            "showInfo" true
+            "class" "mb--2xl contributors--has-no-margin-top"
+            "title" (i18n "about.alumni.title")
+            "titleSize" "h4"
+            "description" (i18n "about.alumni.subtitle")
+          ) }}
         </div>
         <div>
-          <h3>{{ i18n "about.community_managers.title" }}</h3>
-          <p>{{ i18n "about.community_managers.subtitle" }}</p>
-          {{ partial "contributors.html" (dict "staffType" "communityManagers" "showInfo" true "class" "mb--2xl contributors--has-no-margin-top" ) }}
+          {{ partial "contributors.html" (dict
+            "staffType" "communityManagers"
+            "showInfo" true
+            "class" "mb--2xl contributors--has-no-margin-top"
+            "title" (i18n "about.community_managers.title")
+            "description" (i18n "about.community_managers.subtitle")
+          ) }}
         </div>
         <div>
-          <h3>{{ i18n "about.bizdev.title" }}</h3>
-          <p>{{ i18n "about.bizdev.subtitle" }}</p>
-          {{ partial "contributors.html" (dict "staffType" "bizDev" "showInfo" true "class" "mb--2xl contributors--has-no-margin-top" ) }}
+          {{ partial "contributors.html" (dict
+            "staffType" "bizDev"
```

Comment:
Can we swap the CM and BizDev order? It seems to me more relevant to have BizDev closer to Core Team and CM closer to Contributors :)